packages/opentelemetry-exporter-prometheus/src/PrometheusExporter.ts

```diff
@@ -157,6 +157,18 @@ export class PrometheusExporter implements MetricExporter {
     });
   }
 
+  /**
+   * Request handler that responds with the current state of metrics
+   * @param request Incoming HTTP request of server instance
+   * @param response HTTP response objet used to response to request
+   */
+  public getMetricsRequestHandler(
```

Comment:
Note: I'm not a Prometheus user and have no knowledge of it's inner workings, but I think the name and description of this function is misleading based on what the issue is talking about and looking at the implementation.

Based on m limited understanding it looks like you are really trying to "reuse" the internal pipeline, rather than "return" (i.e. get) the MetricsRequestHandler.

As such I would think (I'll let the Prometheus experts and approvers / maintainers correct me and tell me I'm wrong) the name of the method would better be named as "handleRequest" or "publishMetrics" or something like that based on the way the internal _exportMetrics() function is used from the _requestHandler. There may also be some considerations around the batcher usage if the request is a different URL's...